internal/common/environment.go

```diff
@@ -19,3 +25,30 @@ func GetTemplateValidatorImage() string {
 	const defaultImage = "quay.io/kubevirt/kubevirt-template-validator"
 	return envOrDefault(templateValidatorImage, defaultImage+":"+defaultVersion)
 }
+
+func GetNodeLabellerImage(name string) string {
```

Comment:
I would change this function to `GetNodeLabellerImages() NodeLabellerImages` and have it return this type:
```
type NodeLabellerImages struct {
    NodeLabeller string
    Sleeper string
    KVMInfoNFD string
    CPUNFD string
    VirtLauncher string
}
```
WDYT?

Reply:
I changed the function as suggested, the entire struct is set and returned every time it is called as opposed to just returning one specific container image.